applications/FemToDemApplication/custom_processes/stress_to_nodes_process.hpp

```diff
@@ -165,14 +160,12 @@ class StressToNodesProcess : public Process
         rmax_id = aux;
     }
 
-    //----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
-
   protected:
     // Member Variables
-    ModelPart &mr_model_part;
+    ModelPart& mr_model_part;

```

Comment:
Should be mrModelPart

Reply:
done thx

Reply:
Done¿?

Reply:
I was waiting for more things before pushing hehe